### ISSUE TITLE
Downloaded files are empty 0kb

### ISSUE THREAD

**Issue by axelcocaine** [jcgruenhage/PlayMusicExporter#35]
Every playlist, album or song just creates an empty file, I'm on Nougat with Magisk, help!!

**Comment by jcgruenhage** [comment]
I've heard some report of that, can't reproduce it yet though. In v0.9.7 I'll add some checks whether the file was exported successfully, and if not, examine what is special about the environment.

**Comment by axelcocaine** [comment]
Any idea of when we will have an update coming?

**Comment by jcgruenhage** [comment]
No, I currently have a lot of other stuff to do.. But I'll try to fit it in somewhere. May I ask what ROM you are using? And the version of Play Music? µG or regular GApps?

**Comment by axelcocaine** [comment]
Galaxy s4 CrDroid rom 7.1.2, regular Gapps GPM ver. 7.7.4721-1.Q.3956937

**Comment by jcgruenhage** [comment]
I have that device lying around somewhere, I'll try to reproduce this environment as close as possible. just stock opengapps? Which crdroid build?

**Comment by axelcocaine** [comment]
Just changed to Resurrection Remix 7.1.2, same issue.
Hopefully you can fix it, also it depends on your model, mine is i9505 or *jfltexx"

If you have the same here's the link: https://forum.xda-developers.com/galaxy-s4/i9505-develop/rom-resurrection-remix-v5-8-0-t3507378

**Comment by jcgruenhage** [comment]
That's the international version, right?

**Comment by axelcocaine** [comment]
Yes it is, so I just clean installed same rom again and it finally worked, I think it has something to do wuth Magisk hide mode to enable safetynet, can you test that out so I won't screw my phone again? ):

**Comment by jcgruenhage** [comment]
I was using the app on Lineage with MagiskSU and Magisk hide successfully, so that shouldn't have been the problem. I can test it with the S4 once I get that again, my sister had it with her, and she visits tomorrow, and then I won't have time for at least a week, so you gotta wait a bit.

**Comment by axelcocaine** [comment]
Absolutely, I really appreciate your work and support! Thank u so much!

**Comment by jcgruenhage** [comment]
no problem :) Let's just hope it keeps on working for you ^^

**Comment by adminmip** [comment]
So I understand that this is a dead issue, but I'm on an LG G3, VS985 on LineageOS with MagiskSU and Magisk Hide.  Same exact issue.  All I had to do was flip MagiskHide to off, and for whatever reason, boom, fixed.  Still though, thank you for your work!

**Comment by jcgruenhage** [comment]
Does anyone still have problems with this?
